modules/instrument_builder/js/instrument_builder.instrument.js

```diff
@@ -224,64 +227,74 @@ var Instrument = {
-                        $("#" + pieces[0]).click(); break;
-                        break;
-                        
-                }
+                    }
+                    switch(pieces[0]) {
+                        case "table":
+                            document.getElementById("filename").value = pieces[1]; continue;
+                        case "title":
+                            document.getElementById("longname").value = pieces[1]; continue;
+                        case "text":
+                            $("#textbox").click(); break;
+                        case "selectmultiple":
+                            $("#multiselect").click();
+                            //set type to multi for case of multiselect
+                            type = "multi"
+                            ParseSelectOptions(pieces[3], type);
+                            break;
+                        case "select":
+                            $("#dropdown").click();
+                            ParseSelectOptions(pieces[3]);
+                            break;
+                        case "header":
+                            // lots of things are saved as "header".. need to do
+                            // a little detective work
+                            if(pieces[1]) {
+                                $("#scored").click(); break;
+                            }
+                            $("#header").click(); break;
+                        case "static":
+                            if(pieces[1]) {
+                               $("#scored").click(); break;
+                            } else {
+                               if(pieces[2] == '<br />') {
+                                   $("#line").click(); break;
+                               } else {
+                                   $("#label").click(); break;
+                               }
+                            }
+                        case "page":
+                            $("#page-break").click(); break;
+                        default:
+                            $("#" + pieces[0]).click(); break;
+                            break;
+                    }
 
-                document.getElementById("questionName").value = pieces[1];
-                document.getElementById("questionText").value = pieces[2];
-                addQuestion();
-                clearDropdownOption();
-                document.getElementById("questionName").value = '';
-                document.getElementById("questionText").value = '';
+                    document.getElementById("questionName").value = pieces[1];
+                    document.getElementById("questionText").value = pieces[2];
+                    addQuestion();
+                    /*
+                     * pass the type to the clearDropdownOption function to remove
+                     * the options in the correct select element
+                     */
+                    clearDropdownOption(type);
```

Comment:
Adding argument type to clearDropdownOption() call.  
This is the key change Jordan made so that multiselect options will load only for the relevant question.